android/src/main/AndroidManifest.xml

```diff
@@ -67,6 +67,7 @@
                 <category android:name="android.intent.category.BROWSABLE" />
 
                 <data android:scheme="http" />
+                <data android:scheme="https" />
```

Comment:
SO says this [will work](https://stackoverflow.com/a/45782550/4548500), but could you test this with your build already setup just to make sure?